play/src/components/RoundTextIcon.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import SelectedAnswerImage from '../img/selectedAnswerImage.svg';
+import UnselectedAnswerImage from '../img/unselectedAnswerImage.svg';
+import CorrectAnswerImage from '../img/correctAnswerImage.svg';
+
+interface RoundTextIconProps {
+  answerStatus: 'default' | 'correct' | 'selected';
+  index: number;
+  answerText: string;
+  isSubmitted: boolean;
+  handleSelectAnswer: (index: number) => void;
+}
+
+const RoundTextIcon = ({
+  answerStatus,
+  index,
+  answerText,
+  isSubmitted,
+  handleSelectAnswer,
+} : RoundTextIconProps) => {
+  const classes = useStyles(isSubmitted)();
+  const letterCode = 'A'.charCodeAt(0) + index; 
+
+  const imageMap = {
+    'default': UnselectedAnswerImage,
+    'correct': CorrectAnswerImage,
+    'selected': SelectedAnswerImage,
+  }
+
+  const styleMap = {
+    'default': classes.buttonDefault,
+    'correct': classes.buttonCorrect,
+    'selected': classes.buttonSelected,
+  }
+ 
+  return (
+    <div className={classes.container}>
+      <Button
+        className={`${classes.buttonBase} ${styleMap[answerStatus]}`}
+        onClick={() => handleSelectAnswer(index)}
+        disabled={isSubmitted || answerStatus === 'correct'}
```

Comment:
Instead of repeating `'correct'` string in this file, probably better to create a local variable to assign it to `'correct'` and then use that. Even better, use enum to define answer status options and then you can even assign strings to each case. That way, it's both strongly typed, and also has a value associated to each case. [This doc](https://www.typescriptlang.org/docs/handbook/enums.html) has an example for it. I believe I'm also using this mechanism in networking for `GameSessionState`

Reply:
Nice, lin\PlayModels.tsx has been created with an enum for this purpose